railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_1.rb.tt

```diff
@@ -25,43 +30,59 @@
 #   "Referrer-Policy" => "strict-origin-when-cross-origin"
 # }
 
+###
 # Do not treat an `ActionController::Parameters` instance
 # as equal to an equivalent `Hash` by default.
+#++
 # Rails.application.config.action_controller.allow_deprecated_parameters_hash_equality = false
 
+###
 # Active Record Encryption now uses SHA-256 as its hash digest algorithm. Important: If you have
 # data encrypted with previous Rails versions, there are two scenarios to consider:
 #
 # 1. If you have +config.active_support.key_generator_hash_digest_class+ configured as SHA1 (the default
 # before Rails 7.0), you need to configure SHA-1 for Active Record Encryption too:
+#++
 # Rails.application.config.active_record.encryption.hash_digest_class = OpenSSL::Digest::SHA1
+
+###
```

Comment:
I think this line should be removed, because this whole block is one config (`hash_digest_class`) and you need to either pick the config above this line or the one below it. Currently it implies you can choose both.

Reply:
Yeah. It is the same with `support_sha1_for_non_deterministic_encryption`. Not sure how to best denote that this new framework default requires choosing between tree configs. Let group them again and see how it looks. Can you update the branch yourself please?